src/cast.rs

```diff
@@ -572,6 +631,85 @@ impl_from_primitive!(u128, to_u128);
 impl_from_primitive!(f32, to_f32);
 impl_from_primitive!(f64, to_f64);
 
+macro_rules! impl_from_primitive_nonzero {
+    ($T:ty, $to_ty:ident) => {
+        #[allow(deprecated)]
```

Comment:
I think this is also vestigial, way back from commit 03db5c9d1b7cc8625b507184ed3ecba9155b8cb8 when it was copied from its deprecated state in the standard library.

Reply:
Done